tests/test_roomba_integration.py

```diff
@@ -65,7 +65,7 @@ async def test_roomba_connect_error(self, broker, event_loop):
         assert not is_connected
 
     async def roomba_connect(self, roomba, loop):
-        await loop.run_in_executor(None, roomba.connect)
+        await loop.run_in_executor(None, roomba.tr)
```

Comment:
I may be blind.

`tr`?
Where's that defined?

Reply:
oops, right